eth2/operation_pool/src/lib.rs

```diff
@@ -135,7 +139,7 @@ impl<T: EthSpec> OperationPool<T> {
                 )
                 .is_ok()
             })
-            .map(|att| AttMaxCover::new(att, earliest_attestation_validators(att, state)));
+            .map(|att| AttMaxCover::new(att, state, spec, total_active_balance));
```

Comment:
I'd prefer if the `spec` argument came at the end, as it traditionally does :smile: